extensions/wikia/TemplateClassification/services/UserTemplateClassificationService.class.php

```diff
@@ -35,12 +44,37 @@ class UserTemplateClassificationService extends TemplateClassificationService {
 	 * @throws \Swagger\Client\ApiException
 	 */
 	public function getType( $wikiId, $pageId ) {
-		$templateType = parent::getType( $wikiId, $pageId );
+		return $this->getMappedType( parent::getType( $wikiId, $pageId ) );
```

Comment:
Since you call `UserTemplateClassificationService` anyway I think that `getMappedType` reflects the purpose of this method better. :)